liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy

```diff
@@ -0,0 +1,132 @@
+package com.liferay.blade.tests
+
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+import spock.lang.Specification
+import de.undercouch.gradle.tasks.download.Download;
+import java.util.jar.JarInputStream;
+import java.io.FileNotFoundException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+import aQute.bnd.osgi.Jar;
+
+class BladeTests extends Specification {
+	public void executeBlade(String[] args) {
+		"java -jar build/${bladeclijar} args".execute()
+	}
+
+	def setup () {
+	    private static String getLatestRemoteBladeCLIJar() throws BladeCLIException
```

Comment:
this method should go outside of setup().  Methods should never be defined in other methods.  (unless its an anoymous function() :) we can discuss later)